applications/nrf_desktop/README.rst

```diff
@@ -860,28 +895,33 @@ Moreover, keep the default ``ZDebug`` build type that the application is built w
 .. note::
     The following procedure uses the gaming mouse configuration as an example.
 
-To use the nRF Desktop project with your custom board:
+To use the nRF Desktop application with your custom board:
 
-1. Define the board by copying the nRF Desktop board files that are the closest match for your hardware.
-   For example, for gaming mouse use ``nrf/boards/arm/nrf52840gmouse_nrf52840``.
+1. Define the reference design by copying the nRF Desktop reference design files that are the closest match for your hardware.
```

Comment:
board or reference design in this paragraph?
Zephyr uses `board` for that case

Reply:
We will solve this in Pawel's PR. Need to double check.